docs/acme-divergences.md

```diff
@@ -46,6 +46,8 @@ Boulder does not implement the `meta` field returned by the `directory` endpoint
 
 Boulder does not implement the `terms-of-service-agreed` or `orders` fields in the registration object (nor the endpoints the latter links to).
 
+Boulder does not return the `status` field.
```

Comment:
> 6.1.2 defines the account resource with "status (required, string)". That's why I have chosen this section.

That's fair - Boulder does implement the field which is why I thought it might be a better fit for 6.3 since it was a question of whether or not the field is returned. I think overall it might be a better location, could you update accordingly? 

> But imho what should be returned on POSTing is not covered by the standard. I tried to explain this on the ACME list. But I am pretty sure it should be an account resource (since it has not other use in the standard).

I agree - I think your reading that an account resource should be returned is correct. I didn't see your mailing list post in the volley of activity lately but it raises some good feedback! I will try to reply - I think the spec should address the feedback.